tests/models/test_taskinstance.py

```diff
@@ -855,6 +855,25 @@ def test_check_task_dependencies(self, trigger_rule, successes, skipped,
         self.assertEqual(completed, expect_completed)
         self.assertEqual(ti.state, expect_state)
 
+    @parameterized.expand([
```

Comment:
There was another test in the original PR:

```python
    def test_respects_prev_dagrun_dep(self):
        dag = models.DAG(dag_id='test_dag')
        task = DummyOperator(dag=dag, task_id='test_task', start_date=DEFAULT_DATE)
        ti = TI(task, DEFAULT_DATE)
        failing_status = [TIDepStatus('test fail status name', False, 'test fail reason')]
        passing_status = [TIDepStatus('test pass status name', True, 'test passing reason')]
        with patch('airflow.ti_deps.deps.prev_dagrun_dep.PrevDagrunDep.get_dep_statuses',
                   return_value=failing_status):
            self.assertFalse(ti.are_dependencies_met())
        with patch('airflow.ti_deps.deps.prev_dagrun_dep.PrevDagrunDep.get_dep_statuses',
                   return_value=passing_status):
            self.assertTrue(ti.are_dependencies_met())
```

Any reason why we didn't include it?

Reply:
Ah good point there. I somehow reasoned this wasn't necessary. I couldn't quite remember what the exact reason was. But it could be a misjudgement because now that I take a second look at it, I think we should add it in. Thanks for catching that!

Reply:
Added! :) I really couldn't remember why I omitted it